lti_emailer/views.py

```diff
@@ -56,4 +59,11 @@ def lti_launch(request):
         "lti_emailer launched with params: %s",
         json.dumps(request.POST.dict(), indent=4)
     )
-    return redirect('mailing_list:index', request.POST['resource_link_id'])
+
+    view = 'mailing_list'
+    if has_lti_role(request, const.LEARNER):
```

Comment:
The value of the 'roles' parameter in the LTI launch provides the list of roles you have been assigned at the account level, plus the roles you have been assigned for the given course context. I think the logic that Dave has suggested using the roles parameter value is what we want here.